Area51/Sources/ContributionsController.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+
+class ContributionController: UITableViewController {
+
+    var listOfContributors = [Contributor]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UINib(nibName: "ContributorCell", bundle: nil), forCellReuseIdentifier: "ContributorCell")
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        DispatchQueue.global(qos: .background).async {
+            self.getContributors { (res) in
+                switch res {
+                case .success(let contributors):
+                    contributors.forEach({ (contributor) in
+                        self.listOfContributors.append(contributor)
+                    })
+                case .failure(let err):
+                    print(err)
+                }
+            }
+        }
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        DispatchQueue.main.async {
+             self.tableView.reloadData()
+        }
+    }
+
+    fileprivate func getContributors(completion: @escaping (Result<[Contributor], Error>) -> Void) {
+        let request = "https://api.github.com/repos/kgellci/Area51/contributors?anon=1"
```

Comment:
Would be nice to move this functionality to a networking layer. We can create a GitHub issue which someone can pickup for a follow up refactor PR.